src/session/session_api.c

```diff
@@ -648,8 +648,16 @@ __wt_open_cursor(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner, co
 
     hash_value = 0;
 
-    /* We should not open other cursors when there are open history store cursors in the session. */
-    WT_ASSERT(session, strcmp(uri, WT_HS_URI) == 0 || session->hs_cursor_counter == 0);
+    /*
+     * We should not open other cursors when there are open history store cursors in the session.
+     * There are two exceptions to this rule:
+     *  - Verifying the metadata through an internal session and,
```

Comment:
np: Don't need the `and,` since you are doing dotpoints

Reply:
Fixed in e6b398cf27024695b97fbd63ee572437ddd67269